mage/command_string.go

```diff
@@ -2,15 +2,15 @@
 
 package mage
 
-import "fmt"
+import "strconv"
```

Comment:
interesting, we must be using different versions of stringer.

Reply:
Ah, I was using `71657689` for `golang.org/x/tools/cmd/stringer`, I just updated to the latest `master`, `ac136b6c`, and see the same code generated, so I assume this is what `stringer` is generating right now.